pre-and-post-processing/gmsh/README.md

```diff
@@ -0,0 +1,46 @@
+# Gmsh
+## Pre-Processing For 3D Tetrahedral Mesh
+This folder contains sample gmsh input and conversion files, useful for
+mpm pre-processing with tetrahedral elements, as described in the [MPM wiki](https://github.com/geomechanics/mpm/wiki/Gmsh).
+Additional example files are provided for reference: to check that the process
+can be correctly replicated.
+
+**Steps:**
+* Import the gmsh input file (herein [sample_gmsh_input_tetrahedron.geo](https://github.com/geomechanics/mpm-libraries/pre-and-post-processing/gmsh/sample_gmsh_input_tetrahedron.geo)) to gmsh.
```

Comment:
I think you'll need to include `"blob/branch-name/"` for the links to work. 

```suggestion
* Import the gmsh input file (herein [sample_gmsh_input_tetrahedron.geo](https://github.com/geomechanics/mpm-libraries/blob/main/pre-and-post-processing/gmsh/sample_gmsh_input_tetrahedron.geo)) to gmsh.
```

Reply:
See geomechanics/mpm-example README -> https://raw.githubusercontent.com/geomechanics/mpm-examples/main/README.md